projects/documentation/content/deprecation.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: deprecation.njk
+title: 'Deprecation Guide: Spectrum Web Components'
+displayName: Deprecation Guide
+slug: deprecation-guide
+---
+
+# Deprecation Guidelines
```

Comment:
```suggestion
# Deprecation guidelines
```
Spectrum expects this.

Separately, is this `Deprecation process`?

Reply:
Changed this to say `Introduction` instead. There is a heading of `Deprecation process` in the later half!